src/store/FilingHistoryList/getters.ts

```diff
@@ -77,5 +77,23 @@ export default {
   /** Whether one document is downloading. */
   isLoadingOne (state: FilingHistoryListStateIF): boolean {
     return state.loadingOne
+  },
+
+  isAuthorizedToContinueOut (state: FilingHistoryListStateIF, rootGetters: any): boolean {
+    const ccoFilings = state.filings.filter(val => {
```

Comment:
Consider using [find](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find) instead -- it will return the first match (or undefined, which is falsy).

Reply:
Looks good.